product_docs/docs/pgd/5/durability/group-commit.mdx

```diff
@@ -108,5 +123,49 @@ transaction abort is requested. If the transaction is already
 decided to be committed at the time the abort request is sent, the transaction
 does eventually COMMIT even though the client might receive an abort message.
 
+See also [Limitations](limitations). 
+
+### Transaction Reconciliation
+
+A group commit transaction’s commit on the origin node is implicitly converted 
+into a two step commit. 
+
+In the first step the transaction is prepared locally and made ready to commit.
+The data is made durable but uncomitted at this stage so other transactions
+cannot see the changes made by this transaction. This prepared transaction gets
+copied to all remaining nodes through normal replication. 
```

Comment:
logical replication

Reply:
```suggestion
copied to all remaining nodes through normal logical replication. 
```
The purpose of the normal phrasing is to emphasize that this is not an out of band/fast track replication (which people have assumed)